MarbleRally/2016/2016-cumulative-points.csv

```diff
@@ -0,0 +1,34 @@
+Racer,Race 1,Race 2,Race 3,Race 4,Race 5,Race 6,Race 7,Race 8,Race 9,Race 10,Race 11,Race 12
+CMT,0,0,0,0,0,14,17,18,22,27,41,47
+RTA,8,15,16,29,40,40,52,63,68,84,84,84
+SRT,14,26,32,44,44,44,62,75,82,89,89,89
+GSY,4,12,26,35,51,51,62,68,68,76,87,94
+BSK,12,25,25,25,38,38,43,45,64,70,78,96
+H2B,11,16,16,16,28,36,40,61,70,74,99,99
+SBL,0,0,10,31,39,39,39,49,63,88,101,101
+WOD,9,23,44,44,46,56,64,71,82,82,92,102
+FSY,22,22,22,32,32,39,49,57,70,81,102,118
+TTA,7,7,19,44,69,70,72,75,87,100,108,121
+TFS,0,21,46,46,50,65,72,77,93,108,108,123
+SUM,0,0,0,0,9,39,52,70,78,78,96,126
+POL,30,41,54,72,74,74,95,120,135,135,141,141
+SMA,5,9,14,30,45,45,59,74,104,118,133,142
+MMM,1,17,33,39,69,78,87,101,104,114,119,144
+ELC,15,45,53,55,55,58,64,70,72,93,123,144
+SLR,13,13,13,16,21,42,79,88,106,136,145,157
+DRA,18,36,66,81,102,102,117,129,139,148,160,171
+BPL,21,36,45,59,59,70,95,125,131,149,165,173
+GHO,25,50,68,98,104,104,120,136,161,173,180,194
```

Comment:
Could you put these results in descending order, with the most points at the top?